graderx/graders/import_submissions.py

```diff
@@ -286,8 +288,9 @@ def import_submissions(self):
 
         iterCol = iter(submission_list)
```

Comment:
python uses `snake_case` naming.

`URL[0]` isn't really helpful :sweat_smile:

Reply:
about URL[0], you mean the naming ( shouldn't be capital letters ) or about the ambiguity of what I'm trying to get from URL

Reply:
You can use uppercase letters if those are constants, and yes the name is ambiguous